ophyd/sim.py

```diff
@@ -913,6 +915,95 @@ def make_fake_device(cls):
     return fake_device_cache[cls]
 
 
+def clear_fake_device(dev, *, default_value=0, default_string_value='',
+                      ignore_exceptions=True):
+    '''Clear a fake device by setting all signals to a specific value
+
+    Parameters
+    ----------
+    dev : Device
+        The fake device
+    default_value : any, optional
+        The value to put to non-string components
+    default_string_value : any, optional
+        The value to put to components determined to be strings
+    ignore_exceptions : bool, optional
```

Comment:
Why did we need this?

Reply:
Are we guaranteed `sim_put` will exist on all faked signals and that it will never raise?

My thought was: broadcast "clear" values where possible, ignoring anything that might fail.
If it's not a concern, I can easily nuke it.

Reply:
`sim_put` will exist on all faked signals, but we are not guaranteed it will never raise

Reply:
I flipped the default `ignore_exeptions` to `False`, and given that we're guaranteed `sim_put` to exist I'm not including a `hasattr` check.